packages/next/src/server/dev/hot-reloader-webpack.ts

```diff
@@ -852,13 +852,19 @@ export default class HotReloaderWebpack implements NextJsHotReloaderInterface {
                   isDev: true,
                   page,
                 })
-              : {}
-
-            if (staticInfo.amp === true || staticInfo.amp === 'hybrid') {
-              this.hasAmpEntrypoints = true
+              : undefined
+
+            if (staticInfo?.type === PAGE_TYPES.PAGES) {
+              if (
+                staticInfo.config?.config?.amp === true ||
```

Comment:
how come this is a doubly nested config?

Reply:
`pages/` supports the following:

```
export const config = { runtime: "edge" }
```